internal/db/db_blueprints.go

```diff
@@ -0,0 +1,68 @@
+package db
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"github.com/google/uuid"
```

Comment:
nitpick, space between built-in imports and external imports.

Reply:
Fixing too.